staff_features/resources/step_definitions/resource_tree_navigate.rb

```diff
@@ -0,0 +1,141 @@
+  expect(page).to have_text "Resource Resource #{@uuid} created"
+
+  url_parts = current_url.split('/')
+  url_parts.pop
+  @resource_id = url_parts.pop
+
+  click_on 'Add Child'
+  wait_for_ajax
+  fill_in 'Title', with: "Archival Object 1 #{@uuid}"
+  select 'Class', from: 'Level of Description'
+  click_on 'Save'
+  expect(page).to have_text "Archival Object Archival Object 1 #{@uuid} on Resource Resource #{@uuid} created"
+
+  click_on 'Add Child'
+  wait_for_ajax
+  fill_in 'Title', with: "Archival Object 2 #{@uuid}"
+  select 'Class', from: 'Level of Description'
+  click_on 'Save'
+  expect(page).to have_text "Archival Object Archival Object 2 #{@uuid} created as child of Archival Object 1 #{@uuid} on Resource Resource #{@uuid}"
+  wait_for_ajax
+end
+
+Given 'the Resource is opened in edit mode' do
+  visit "#{STAFF_URL}/resources/#{@resource_id}/edit"
+
+  wait_for_ajax
+end
+
+When 'the user opens the Resource in edit mode' do
+  visit "#{STAFF_URL}/resources/#{@resource_id}/edit"
+
+  wait_for_ajax
+end
+
+Then 'the Resource is displayed as the top level of the navigation tree' do
+  rows = all('#tree-container .table .table-row', visible: :all)
+
+  expect(rows.length).to eq 4
+  expect(rows[0].text).to include "Resource #{@uuid}"
+end
+
+Then 'the tree includes the two Archival Objects' do
+  rows = all('#tree-container .table .table-row')
+
+  expect(rows.length).to eq 2
+  expect(rows[2].text).to include "Archival Object 1 #{@uuid}"
+end
+
+Then 'the Resource is highlighted in the tree' do
+  rows = all('#tree-container .table .table-row')
+
+  expect(rows.length).to eq 2
+  expect(rows[0].text).to include "Resource #{@uuid}"
+  expect(rows[0][:class]).to include 'current'
+end
+
+Given 'only the first-level Archival Objects are displayed' do
+  rows = all('#tree-container .table .table-row')
+
+  expect(rows.length).to eq 2
+  expect(rows[1].text).to include "Archival Object 1 #{@uuid}"
+end
+
+Then 'the button has text {string}' do |button_text|
+  expect(page).to have_text button_text
+end
+
+Then 'the expand arrows are disabled' do
+  arrows = all('.expandme')
+
+  arrows.each do |arrow|
+    expect(arrow[:class]).to include 'disabled'
+  end
+end
+
+Then 'all Archival Objects are displayed' do
+  wait_for_ajax
+
+  rows = all('#tree-container .table .table-row')
+
+  expect(rows.length).to eq 3
+  expect(rows[1].text).to include "Archival Object 1 #{@uuid}"
+  expect(rows[2].text).to include "Archival Object 2 #{@uuid}"
+end
+
+Then 'only the top-level Archival Objects are displayed' do
+  rows = all('#tree-container .table .table-row')
+
+  loop do
+    break if rows.length == 2
+
+    sleep 1
+    rows = all('#tree-container .table .table-row')
+  end
+
+  expect(rows.length).to eq 2
+  expect(rows[1].text).to include "Archival Object 1 #{@uuid}"
+end
+
+Given 'all levels of hierarchy in the tree are expanded' do
+  click_on 'Auto-Expand All'
+
+  wait_for_ajax
+end
+
+Then 'the expand arrows are enabled' do
```

Comment:
see earlier comments.  test is whether expand arrows don't have the disabled class

Reply:
Same as above; whatever we decide should be also applied here.